spectator/registry.cc

```diff
@@ -26,7 +26,19 @@ auto Registry::GetLogger() const noexcept -> Registry::logger_ptr {
 }
 
 auto Registry::GetAgeGauge(Id id) noexcept -> std::shared_ptr<AgeGauge> {
-  return all_meters_.insert_age_gauge(std::move(id));
+  if (all_meters_.age_gauges_.contains(id)) {
+    return all_meters_.age_gauges_.at(id);
+  } else {
+    if (all_meters_.age_gauges_.size() < config_->age_gauge_limit) {
+      return all_meters_.insert_age_gauge(std::move(id));
+    } else {
+      logger_->warn("maximum number of age gauges (" +
+                    std::to_string(config_->age_gauge_limit) +
```

Comment:
No need to convert to string explicitly.

Reply:
I am not sure how else this would be written, `std::format` maybe?

Reply:
`spdlog` forwards all arguments to `fmt::format` so integers, etc. can be used directly.